src/futures/account.rs

```diff
@@ -0,0 +1,252 @@
+    }
+
+    fn build_order(&self, order: OrderRequest) -> BTreeMap<String, String> {
+        let mut parameters = BTreeMap::new();
+        parameters.insert("symbol".into(), order.symbol);
+        parameters.insert("side".into(), order.side.into());
+        parameters.insert("type".into(), order.order_type.into());
+
+        if let Some(position_side) = order.position_side {
+            parameters.insert("positionSide".into(), position_side.into());
+        }
+        if let Some(time_in_force) = order.time_in_force {
+            parameters.insert("timeInForce".into(), time_in_force.into());
+        }
+        if let Some(qty) = order.qty {
+            parameters.insert("quantity".into(), qty.to_string());
+        }
+        if let Some(reduce_only) = order.reduce_only {
+            parameters.insert("reduceOnly".into(), reduce_only.to_string().to_uppercase());
+        }
+        if let Some(price) = order.price {
+            parameters.insert("price".into(), price.to_string());
+        }
+        if let Some(stop_price) = order.stop_price {
+            parameters.insert("stopPrice".into(), stop_price.to_string());
+        }
+        if let Some(close_position) = order.close_position {
+            parameters.insert("closePosition".into(), close_position.to_string().to_uppercase());
+        }
+        if let Some(activation_price) = order.activation_price {
+            parameters.insert("activationPrice".into(), activation_price.to_string());
+        }
+        if let Some(callback_rate) = order.callback_rate {
+            parameters.insert("callbackRate".into(), callback_rate.to_string());
+        }
+        if let Some(working_type) = order.working_type {
+            parameters.insert("workingType".into(), working_type.into());
+        }
+        if let Some(price_protect) = order.price_protect {
+            parameters.insert("priceProtect".into(), price_protect.to_string().to_uppercase());
+        }
+
+        parameters
+    }
+
+    pub fn change_position_mode(&self, dual_side_position: bool) -> Result<()> {
+        let mut parameters: BTreeMap<String, String> = BTreeMap::new();
+        let dual_side = if dual_side_position { "true" } else { "false" };
+        parameters.insert("dualSidePosition".into(), dual_side.into());
+
+        let request = build_signed_request(parameters, self.recv_window)?;
+        self.client
+            .post_signed::<Empty>(API::Futures(Futures::PositionSide), request)
+            .map(|_| ())
+    }
+
+    pub fn cancel_all_open_orders<S>(&self, symbol: S) -> Result<()> where S: Into<String> {
+        let mut parameters: BTreeMap<String, String> = BTreeMap::new();
+        parameters.insert("symbol".into(), symbol.into());
+        let request = build_signed_request(parameters, self.recv_window)?;
+        self.client
+            .delete_signed::<Empty>(API::Futures(Futures::AllOpenOrders), Some(request))
+            .map(|_| ())
+    }
+}
+
+
```

Comment:
This is missing the `#[cfg(test)]`. Also, the tests are just stubs and are failing.